README.md

```diff
@@ -1,207 +1,229 @@
 
-It is recommended to use the [STAC Processing Extension][stac-ext-proc]
-to represent the `processing:level` of the relevant level `L0` for raw data up to `L4` for Analysis-Ready Data (ARD).
+#### Parameters Object
 
-[stac-ext-proc]: https://github.com/stac-extensions/processing#suggested-processing-levels
+| Field Name                            | Type                         | Description                                                                                                                                                                                                                                                                      |
+|---------------------------------------|------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| *parameter names depend on the model* | number `\|` string `\|` boolean `\|` array | The number of fields and their names depend on the model. Values should not be n-dimensional array inputs. If the model input can be represented as an n-dimensional array, it should instead be supplied as another [model input object](#model-input-object). |
 
-#### Data Type Enum
+The parameters field can either be specified in the [Model Input Object](#model-input-object) if they are associated with a specific input or as an [Item or Collection](#item-properties-and-collection-fields) field if the parameters are supplied without relation to a specific model input.
 
-It is recommended to use the [STAC Raster Extension - Raster Band Object][stac-ext-raster-band-obj]
-in STAC Collections and Items that refer to a STAC Item using `dlm`'s `data_type`. The values should be one of the known
-data types defined by `raster:bands`'s `data_type` as presented in [Data Types][stac-ext-raster-dtype].
+#### Bands and Statistics
 
-If source imagery has different `data_type` values than the one specified by `dlm`'s `data_type` property,
-this should provide
-an indication that the source imagery might require a preprocessing step (scaling, normalization, conversion, etc.)
-to adapt the samples to the relevant format expected by the described model.
+We use the [STAC 1.1 Bands Object](https://github.com/radiantearth/stac-spec/pull/1254) for representing bands information, including the nodata value, data type, and common band names. Only bands used to train or fine tune the model should be included in this `bands` field.
 
-[stac-ext-raster-dtype]: https://github.com/stac-extensions/raster/#data-types
-[stac-ext-raster-band-obj]: https://github.com/stac-extensions/raster/#raster-band-object
```

Comment:
I think the `eo:bands` and/or `raster:bands` should remain valid alternatives until STAC 1.1 is released. See https://github.com/crim-ca/dlm-extension/issues/12

Reply:
Agree, but can STAC 1.1 Bands be the default when a user is using `stac-model` to construct the metadata? Would prefer for there to be one clear main way to do things. Ideally folks don't need to think about the different options since they document the same info. I'm assuming this new version is going to be released soon.

Reply:
I agree it would be preferable to have a single method, but since it's still only in proposal state at the moment, I think depending on it will delay this extension for even longer and limit adoption. It would be a shame to limit MLM adoption because existing catalogs are not yet 1.1 compliant. We also do not have any support in `pystac` and STAC API, so generating an Item with 1.1 Bands will not even be employable currently. Since some catalogs already make use of `eo`/`raster`, we can help create the connections to existing bands definitions for the MLM case. Proper use of current 1.0 extensions is barely starting to be integrated correctly across catalogs, so I doubt they will adopt 1.1 before a while.

Reply:
More details added to https://github.com/rbavery/dlm-extension/pull/2